assets/javascripts/discourse/components/translated-post.gjs

```diff
@@ -31,29 +31,35 @@ export default class TranslatedPost extends Component {
     return this.post.translatedTitle;
   }
 
+  get showTranslation() {
+    return !this.siteSettings.experimental_topic_translation;
+  }
+
   <template>
     <div class="post-translation">
       <ConditionalLoadingSpinner
         class="post-translation"
         @condition={{this.loading}}
         @size="small"
       >
-        <hr />
-        {{#if this.translatedTitle}}
-          <div class="topic-attribution">
-            {{this.translatedTitle}}
+        {{#if this.showTranslation}}
+          <hr />
+          {{#if this.translatedTitle}}
+            <div class="topic-attribution">
+              {{this.translatedTitle}}
+            </div>
+          {{/if}}
+          <div class="post-attribution">
+            {{i18n
+              "translator.translated_from"
+              language=this.post.detectedLang
+              translator=this.siteSettings.translator
```

Comment:
Certainly a succinct setting name! :D I thought this was a mistake, I know you didn't add this and it's too late now anyway, but `translator_provider` or something would be better, it doesn't look like an actual setting ATM.

Reply:
Amazing, thank you!